src/main/java/com/kodilla/ecommercee/dto/UserDto.java

```diff
@@ -15,6 +18,7 @@
     private String password;
     private boolean blocked;
     private String address;
+    private List<UserOrder> userOrders;
```

Comment:
Dto zdecydowanie nie powinno mieć relacji do encji. Jeśli chcesz mieć Usera wraz z listą zamówień to zrób nowe DTO(np., UserWithOrdersDto) i w środku niech ma List<UserOrderDto>

Reply:
usunięte, w przyszłości jak będzie potrzeba to zrobię